ci/jenkins/test.sh

```diff
@@ -289,6 +289,7 @@ function deliver_antrea {
         docker tag "${DOCKER_REGISTRY}/antrea/sonobuoy-systemd-logs:v0.3" "sonobuoy/systemd-logs:v0.3"
     fi
     DOCKER_REGISTRY=${DOCKER_REGISTRY} make
+    DOCKER_REGISTRY=${DOCKER_REGISTRY} make flow-aggregator-ubuntu
```

Comment:
so until now the flow-aggregator image was pulled from the docker registry (instead of building the image from the checked-out repo and using that one)?

Reply:
For Jenkins, yes. I found this issue when debugging the Jenkins tests.